src/PositionManager.sol

```diff
@@ -29,7 +31,8 @@ contract PositionManager is
     Multicall,
     SafeCallback,
```

Comment:
can remove SafeCallback from this list now - thats inherited by BaseActionsRouter